tests/phpunit/tests/block-bindings/register.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Tests for Block Bindings API helper functions.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @since 6.5.0
+ *
+ * @group blocks
+ * @group block-bindings
+ */
+class Tests_Block_Bindings_Register extends WP_UnitTestCase {
+
+	const TEST_SOURCE_NAME       = 'test/source';
+	const TEST_SOURCE_PROPERTIES = array(
+		'label' => 'Test source',
+	);
+
+	/**
+	 * Tear down after each test.
+	 *
+	 * @since 6.5.0
+	 */
+	public function tear_down() {
```

Comment:
I think that we should unregister sources in a `set_up` method as well. Once we add the `post-meta` & `pattern` sources to Core, they will be registered for all the tests. I can add that in https://github.com/WordPress/wordpress-develop/pull/5888/

Reply:
It's only `test_get_all_registered` that would get impacted. Let's see if that becomes an actual issue, or we can simply account for that in the assertion.